commons/src/main/java/org/infinispan/commons/dataconversion/MediaTypeResolver.java

```diff
@@ -0,0 +1,57 @@
+package org.infinispan.commons.dataconversion;
+
+import java.io.BufferedInputStream;
+import java.io.InputStream;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Scanner;
+
+import org.infinispan.commons.util.Util;
+
+/**
+ * Resolve media types for files
+ *
+ * @since 10.1
+ */
+public final class MediaTypeResolver {
+
+   private static final Map<String, String> FILE_MAP = new HashMap<>();
+
+   static {
+      InputStream in = null;
+      BufferedInputStream bis = null;
+      try {
+         in = MediaTypeResolver.class.getClassLoader().getResourceAsStream("mime.types");
+         if (in != null) {
```

Comment:
maybe a error message or something will be nice

Reply:
fixed